docs/t-sql/statements/merge-transact-sql.md

```diff
@@ -84,8 +84,9 @@ MERGE
   
 <target_table> ::=  
 {
-    [ database_name . schema_name . | schema_name . ]  
-  target_table  
+    [ database_name . schema_name . | schema_name . ]  target_table  
+    | @variable
+    | common_table_expression_name
```

Comment:
```suggestion
    [ database_name . schema_name . | schema_name . ]  [ [ AS ] table_alias ]
    | @variable [ [ AS ] table_alias ]
    | common_table_expression_name [ [ AS ] table_alias ]
```

Improving existing syntax with proper `AS alias`. This makes sense to me.  I don't have a problem with this update.